Source/Renderer/CubeMap.js

```diff
@@ -216,13 +218,13 @@ define([
                         magFilter = TextureMagnificationFilter.NEAREST;
                     }
 
-                    sampler = {
+                    sampler = new Sampler({
```

Comment:
Any idea why just don't have `_sampler` defined?  It would clean up some logic here.